test/protocol/projects/hardhat/contracts/semanticTokens/full/SemanticTokens.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.8;
+
+interface TestInterface {}
+
+struct TestStruct {
+    uint256 aNumber;
+    string aString;
+    address anAddress;
+}
+
+contract testContract {
+    event TestEvent(
+        testContract contractAsEventParam, TestInterface interfaceAsEventParam, TestStruct structAsEventParam
+    );
+
+    testContract contractAsMember;
+
+    TestInterface interfaceAsMember;
+
+    TestStruct structAsMember;
+
+    uint256 aNumber = 0x12 + 123;
+
+    string aString = "asdf";
+
+    function testFunction(
+        testContract contractAsFuncParam,
+        TestInterface interfaceAsFuncParam,
+        TestStruct memory structAsFuncParam
+    ) public {
+        testContract contractAsLocalVar;
+
+        TestInterface interfaceAsLocalVar;
+        TestStruct memory structAsLocalVar;
+        contractAsLocalVar;
+        contractAsFuncParam;
+        interfaceAsFuncParam;
+        interfaceAsLocalVar;
+        structAsFuncParam;
+        structAsLocalVar;
```

Comment:
not sure I understand what these lines are testing. These are expression statements, and all not highlighted. Is that correct?

Reply:
Yes, that's correct. They were there because at some point they were incorrectly highlighted, but they shouldn't and we are asserting that

Reply:
I suggest adding comments to indicate which is which.